Automattic/pocket-casts-ios
# End of Year: add prompt

| 📘 Project: #376 |
|:---:|

This PR adds a modal that prompts the user to check their End of Year. Design and iPad are not covered in this task and will be done in a future PR.

<img src="https://user-images.githubusercontent.com/7040243/194940985-9b915d54-562f-41da-9666-88a0cc168bba.png" width="300">


## To test

1. On `FeatureFlag.swift` change `endOfYear` to `true`
2. Run the app
3. ✅ Check that a bottom sheet appears with two buttons: "View My 2022" and "Not Now"
4. ✅ Tap "Not Now" and ensure the bottom sheet is dismissed
5. Re-run the app
6. When the bottom sheet appears, tap "View My 2022"
7. ✅ Ensure a VC is shown in `fullScreen` mode

## Checklist
